### DISCUSSION TITLE
[Question] How Can I force mfsmount to use ipv6?

### DISCUSSION THREAD

**Discussion by Sveeeeeen** [moosefs/moosefs#379] [Q&A]
Hello!

How Can I force mfsmount to use ipv6 to my chunkservers?

Thank you!

**Comment by chogata** [comment]
MooseFS does not support IPv6.

**Comment by Sveeeeeen** [comment]
Thanks for adding the label.
Too bad it's not available yet! Its already 2020 :)

**Comment by rpodgorny** [comment]
so, the question is what could be done to implement ipv6 support?

**Comment by chogata** [comment]
This feature is not in our immediate plans.

**Comment by rpodgorny** [comment]
> This feature is not in our immediate plans.

fair enough. will you at least accept pull requests?

**Comment by chogata** [comment]
At the moment - not likely, because of our development schedule and planned changes. In the future - only, if the code meets our standards and keeps backward compatibility (all needed protocol changes must take into account that older modules might not "understand" new protocol - situation, when there is a master that understands IPv6 and a mix of chunkservers and clients that do and don't must be handled correctly).
